releasenotes/notes/deprecate-package-616af0d1ae27e231.yaml

```diff
@@ -0,0 +1,5 @@
+---
+prelude: >
+    ``penaltymodel`` is deprecated and will be removed in Ocean 10.
+    For solving problems with constraints, we recommand using the hybrid
```

Comment:
```suggestion
    For solving problems with constraints, we recommend using the hybrid
```

Reply:
ha! That was wrong in https://github.com/dwavesystems/dwavebinarycsp/pull/124 as well. Will fix. Thanks